tiltak/src/main/kotlin/no/nav/amt/tiltak/tiltak/services/GjennomforingServiceImpl.kt

```diff
@@ -21,7 +24,9 @@ class GjennomforingServiceImpl(
 	private val gjennomforingRepository: GjennomforingRepository,
 	private val tiltakService: TiltakService,
 	private val arrangorService: ArrangorService,
-	private val publisherService: DataPublisherService
+	@Lazy private val deltakerService: DeltakerService,
```

Comment:
Skal tenke litt på om det er bedre å hente deltakerne et annet sted for å unngå denne sirkulære avhengigheten - kanskje det er bedre å gjøre det i `AmtArrangorClient`, men jeg synes ikke det virker like naturlig.

Reply:
Kjedelig med sirkulær avhengighet, men enig i at det er mer logisk å ha dette her